ui/helpers/utils/accounts.test.js

```diff
@@ -101,6 +112,9 @@ describe('Accounts', () => {
         keyring: {
           type: KeyringType.hdKeyTree,
         },
+        snap: {
+          id: 'npm:@metamask/solana-wallet-snap',
```

Comment:
Nit: Same here, we could use `SOLANA_WALLET_SNAP_ID` (and all other occurences).